migration/v0.25.x-v0.26.x/upgrade-apps.md

```diff
@@ -30,3 +47,15 @@
     ```bash
     bin/ck8s apply {sc|wc}
     ```
+
+1. Restore harbor by following the documentation for restoring harbor.
+   Ensure that it is the new config map that is added to the cluster.
+   [Restore documentation](../../scripts/restore/README.md)
+
+1. Login into harbor and check that projects, images, users, robot accounts, etc exists after the restore is completed.
+
+1. Delete old harbor pvc
+
+    ```
+    ./bin/ck8s ops kubectl sc delete pvc -n harbor data-harbor-harbor-redis-0 data-harbor-harbor-trivy-0 database-data-harbor-harbor-database-0 harbor-harbor-jobservice
+    ```
```

Comment:
So harbor will create new PVCs after running apply sc?

Reply:
Yes. This is because harbor has changed how everything is named. Before everything received `<chart name>-harbor-*`. But now they will remove `<chart name>-` if the chart is called harbor. Thus creating new pods and new PVCs and that is also the reason this migration requires a backup and restore.

Reply:
Ok, thanks